index.bs

```diff
@@ -7902,26 +7867,28 @@ To <dfn>get the realm info</dfn> given |environment settings|:
     <dd>
       1. Let |document| be |environment settings|' [=relevant global object=]'s
          <a>associated <code>Document</code></a>.
-      1. Let |browsing context| be |document|'s [=/browsing context=].
-      1. Let |context id| be the [=browsing context id=] for |browsing context|.
+      1. Let |navigable| be |document|'s [=/node navigable=].
+      1. If |navigable| is null, return null.
```

Comment:
You need to update the callers to handle `null` as a return value.

Reply:
I believe most(some) of them do, did you spot one that is not handling it?

Reply:
Found missing places, and updated